sdk/keyvault/keyvault-secrets/test/public/utils/common.ts

```diff
@@ -4,7 +4,12 @@
 import { assert } from "chai";
 import { SupportedVersions, supports, TestFunctionWrapper } from "@azure/test-utils";
 import { env } from "@azure-tools/test-recorder";
-import { LATEST_API_VERSION, SecretClientOptions } from "../../src/secretsModels";
+import { SecretClientOptions } from "../../../src";
+
+/**
+ * The latest supported KeyVault service API version
+ */
+export const LATEST_API_VERSION = "7.3-preview";
```

Comment:
We can infer this from https://github.com/Azure/azure-sdk-for-js/pull/19959/files#diff-c3564ec34ef1be7b6ff757d6352c4dc035b76e9fea338b9dad4298414d2a8b03L27 as well - sorry for repeating myself I just wanted to make sure it doesn't get missed